README.md

```diff
@@ -100,6 +99,12 @@ VALUES (:first_name, :last_name, :country)
 ```
 Notice the query annotations (`--! authors`, `--! insert_authors`) and the named bind parameters (`:first_name`, etc.).
 
+1. First of all, you can organize that queries into folder /queries/ as mentioned or pass the folder when running the CLI
```

Comment:
IMO, a numbered list is not really appropriate here. Perhaps we can simply mention: "By default, Cornucopia reads all query files in the `queries/` folder".